public/cli/lib/commands/ffmpeg.ts

```diff
@@ -4,20 +4,16 @@ import shell from 'shelljs';
 import tempy from 'tempy';
 import path from 'path';
 import { FfmpegSettings } from '../../../models/ffmpegSettings.model';
+import { paths } from '../path-constants'
```

Comment:
Just need to remove the `readdirSorted` import above and this is ready to merge.